pyproject.toml

```
@@ -0,0 +1,38 @@
+[project]
+name = "pep8speaks"
+version = "1.9"
+requires-python = ">=3.8.1"
```

```suggestion
requires-python = ">=3.9.0"
```

as python 3.8 is deprecated.

ref. https://devguide.python.org/versions/